lib/src/component/aria_mixin.dart

```diff
@@ -18,8 +18,7 @@ import 'dart:collection';
 
 // Must import these consts because they are used in the transformed code.
 // ignore: unused_shown_name
-import 'package:over_react/over_react.dart'
-    show PropDescriptor, PropsMeta;
+import 'package:over_react/over_react.dart' show PropDescriptor, PropsMeta;
```

Comment:
Should this be updated to a `src` import?

Reply:
ohh good catch :loading: